packages/foreman/katello-certs-tools/katello-certs-tools.spec

```diff
@@ -12,7 +12,6 @@ Requires: openssl
 Requires: rpm-build
 
 BuildRequires: docbook-utils
-BuildRequires: /usr/bin/pathfix.py
```

Comment:
Why are is this being dropped? There are still shebangs.

Reply:
Only in `katello-ssl-tool` itself, and that gets handled just fine by `%{pyX_build}` which adds `--executable=/usr/bin/python2 -s`. see http://koji.katello.org/kojifiles/work/tasks/6253/176253/build.log for an example